plugins/providers/docker/driver.rb

```diff
@@ -236,8 +236,20 @@ def all_containers
         execute('docker', 'ps', '-a', '-q', '--no-trunc').to_s.split
       end
 
+      # Attempts to first use the docker-cli tool to inspect the default bridge subnet
+      # Falls back to using /sbin/ip if that fails
+      #
       # @return [String] IP address of the docker bridge
       def docker_bridge_ip
+        bridge = inspect_network("bridge")[0]
+        bridge["IPAM"]["Config"][0]["Gateway"]
+      rescue => e
+        @logger.debug("Failed to get bridge ip from docker: #{e}")
+        @logger.debug("Falling back to /sbin/ip")
+        docker_bridge_ip_from_sbin_ip
+      end
+
+      def docker_bridge_ip_from_sbin_ip
         output = execute('/sbin/ip', '-4', 'addr', 'show', 'scope', 'global', 'docker0')
```

Comment:
I think updating this to just `'ip'` and removing the explicit path would be appropriate.